src/app/(simulation)/simulateur/[...dottedName]/_components/Total.tsx

```diff
@@ -0,0 +1,43 @@
+import Link from 'next/link'
+import { useState } from 'react'
+
+import QuestionButton from '@/components/misc/QuestionButton'
+import { useRule } from '@/publicodes-state'
+import Explanation from './total/Explanation'
+import Planet from './total/Planet'
+import Progress from './total/Progress'
+import ValueChangeDisplay from './total/ValueChangeDisplay'
+
+export default function Category() {
+  const { value } = useRule('bilan')
+
+  const [open, setOpen] = useState(false)
```

Comment:
Que penses-tu de la règle d'écriture qui suit pour les booléens : toujours ajouter un préfixe `is`, `has` ou `should` - ici donc on aurait `isOpen` - pour implicitement faire comprendre la nature de booléen de la variable ? D'autre part, en utilisant `open` dans le code, on peut confondre cette variable avec une fonction (verbe d'action). C'est une règle que j'applique depuis un moment et je trouve que ça permet au code de gagner en clarté